{"name": "Проверка корректной ошибки при некорректной авторизации: {test_data_key}", "status": "passed", "parameters": [{"name": "test_data", "value": "{'email': 'karry_karrow_1_111@yandex.ru', 'password': ''}"}], "start": 1733867477455, "stop": 1733867477656, "uuid": "9122b4c0-0981-41f1-95c6-a052c64a91b7", "historyId": "6332b38dc97366ddfea54424488eb670", "testCaseId": "311718a49f0f75c6e906c1fb2ac9a549", "fullName": "tests.test_login user.TestLoginUser#test_login_error_handling", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "Karinas-MacBook-Pro.local"}, {"name": "thread", "value": "8223-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login user"}]}